docs/references/function-list-by-name.md

```diff
@@ -304,7 +304,7 @@ MID                 | CATEGORY_TEXT_AND_DATA         | \PhpOffice\PhpSpreadsheet
 MIDB                | CATEGORY_TEXT_AND_DATA         | \PhpOffice\PhpSpreadsheet\Calculation\TextData::MID
 MIN                 | CATEGORY_STATISTICAL           | \PhpOffice\PhpSpreadsheet\Calculation\Statistical::MIN
 MINA                | CATEGORY_STATISTICAL           | \PhpOffice\PhpSpreadsheet\Calculation\Statistical::MINA
-MINIF               | CATEGORY_STATISTICAL           | \PhpOffice\PhpSpreadsheet\Calculation\Statistical::MINIF
+MINIFS              | CATEGORY_STATISTICAL           | **Not yet Implemented**
```

Comment:
Should indicate that you have implemented MINIFS rather than Not Yet Implemented